net/curl/Portfile

```diff
@@ -192,10 +192,23 @@ if {${name} eq ${subport}} {
                                 path:share/curl/curl-ca-bundle.crt:curl-ca-bundle
         configure.args-replace  --without-ssl --with-ssl=${prefix}
         configure.args-append   --with-ca-bundle=${prefix}/share/curl/curl-ca-bundle.crt
+
+        # remove after 2021-07-01
+        if {${os.platform} eq "darwin"} {
+            notes "
+Please note that the default SSL backend was recently changed from OpenSSL to \
```

Comment:
In notes, don't put a space before the backslash at the end of the line because that will turn into a double space when printed on the terminal.